release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -69,14 +69,24 @@ User-defined tags for Microsoft Azure was previously introduced as Technology Pr
 [id="ocp-4-14-openshift-cli"]
 === OpenShift CLI (oc)
 
-.Supporting multi-arch OCI local images for catalogs with oc-mirror
+[id="oc-mirror-multi-arch-oci-local-images"]
+==== Supporting multi-arch OCI local images for catalogs with oc-mirror
 
 With {product-title} {product-version}, oc-mirror supports multi-arch OCI local images for catalogs.
 
 OCI layouts consist of an `index.json` file that identifies the images held within them on disk. This `index.json` file can reference any number of single or multi-arch images. However, oc-mirror only references a single image at a time in a given OCI layout. The image stored in the OCI layout can be a single-arch image, that is, an image manifest or a multi-arch image, that is, a manifest list.
 
 The `ImageSetConfiguration` stores the OCI images. After processing the catalog, the catalog content adds new layers representing the content of all images in the layout. The ImageBuilder is modified to handle image updates for both single-arch and multi-arch images.
 
+[id="oc-logging-in-browser"]
+==== Logging in to the CLI using a web browser
+
+With {product-title} {product-version}, a new `oc` command line interface (CLI) flag, `--web` is now available for the `oc login` command. 
+
+With this enhancement, users can log in using a web browser which allows users to avoid inserting their access token into the command line.
```

Comment:
Not critical, but I might suggest saying "you" instead of "users" in this sentence.  We probably have a mix of ways right now, but I think it sounds more natural to say "you"

```suggestion
With this enhancement, you can log in using a web browser, which allows you to avoid inserting your access token into the command line.
```